nway/nway_matching.py

```diff
@@ -373,6 +373,9 @@ def run(self):
         """Nway cell matching by calling pairwise
            matching and then combining the results
         """
+        # log this ENV variable, if present
+        commit = os.environ.get("NWAY_COMMIT_SHA", None)
+        logger.info(f"NWAY_COMMIT_SHA {commit}")
```

Comment:
Maybe would hurt to add something along the lines of:

```
import nway
logger.info(f"Nway matching version: {nway.__version__}")
```

Reply:
My thought is that the commit hash is unambiguous, and that maintaining the version on tagging/release (which we're not really doing) is an extra step that is often forgotten, and so what's in `__init__.py` is not reliable.

Reply:
I should update the tag though. Will do when this is merged.

Reply:
@djkapner I guess I favor maintaining a `__version__` just in case in the future there might be software that depends on the nway package. How would that package handle version pinning with a commit SHA?

If that is definitely *never* going to be the case then it would probably make sense to remove the `__version__` info so there is not conflicting information about what the canonical 'version' should be.

Reply:
ok, sounds reasonable. Added that and updated the `__init__.py`